x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/ComputeService.java

```diff
@@ -171,6 +178,13 @@ public void execute(
                 listener.map(driverProfiles -> new Result(collectedPages, driverProfiles))
             );
             return;
+        } else {
```

Comment:
super-nit, feel free to ignore: it could be and `else if {`, no need for another pair of braces